packages/big-design/src/components/Select/styled.tsx

```diff
@@ -19,6 +20,13 @@ export const StyledDropdownIcon = styled(ArrowDropDownIcon)<{ disabled?: boolean
     `}
 `;
 
+export const StyledListGroupHeader = styled(StyledGroupHeader)`
+  font-family: 'Source Sans Pro SemiBold', 'Helvetica Neue', Arial, sans-serif;
```

Comment:
You probably want to use `font-weight` instead of `font-family` to change this type of styling.